src/Psr/Uri.php

```diff
@@ -10,16 +10,16 @@
 use Psr\Http\Message\UriInterface;
 
 /**
- * Class Uri
+ * Class URI
```

Comment:
Refers to the Uri class.

Reply:
Fair, but this is IMHO a rather poor short description doc block because just repeating text that is available as PHP syntax adds not information, thus has no benefits.

A more information rich one would be:

``Unified Resource Link``

I always try to add information, represent the existing information in an alternative way.

I'll revert this for this PR.

If you with, I can open one where I change all class doc blocks like this.

Reply:
PhpStorm used to add this headlines with just the class name and I just kept them and added description below. Yes those headlines with just the class names are maybe not adding value, but the descriptions below do.

An improvement would be to make the headline being a very short summary of the longer description below. If you want to you can give it a try. If you do, please in a separate PR.